package.json

```diff
@@ -5,38 +5,40 @@
   "registerAs": "screenshot",
   "main": "index.js",
   "scripts": {
-    "test": "grunt"
+      "test": "grunt"
```

Comment:
suddenly 4 space indent! (I'm assuming these should have been reverted to the 2-space indents as well?)